### ISSUE TITLE
allow adding locally available SDK

### ISSUE THREAD

**Issue by yjbanov** [cbracken/dvm#2]
Suppose for some crazy reason I built my own version of the SDK available in `/path/to/custom/dart/build`. I should be able to:

`dvm add /path/to/custom/dart/build my_build`

Then `dvm list` would list it as `my_build`.

**Comment by cbracken** [comment]
As a workaround until I get it landed:

```
$ cd $DVM_ROOT/darts
$ ln -s path/to/sdk mysdk
$ dvm use mysdk
```

**Comment by cbracken** [comment]
Closed by a2649a48180fa8cb55e75ae23cf992397928450e.

**Comment by cbracken** [comment]
The above patch allows for:

```
dvm alias create foo --path /path/to/sdk
dvm use foo
```

works with relative paths as well.
